src/couchapps/WMStats/_attachments/js/minified/import-all-t1.min.js

```diff
@@ -4402,7 +4402,6 @@ WMStats.namespace('ResubmissionList');
             htmlstr += "<li><b>ACDC DB Name:</b><input type='text' name='ACDCDatabase' size=30  value='wmagent_acdc'/></li>";
         }
         htmlstr += "<li><b>RequestPriority:</b> " + summary.RequestPriority + "</li>";
-        htmlstr += "<li><b>RequestType:</b> " + summary.RequestType + "</li>";
```

Comment:
This line always gets removed when running combineMinifyWMStats.py, I added it back in the drain stats PR.  Not sure if it is needed or not.

Reply:
Actually, it seems this code was changed manually before. (Not using combinedMinifyWMStats.py) That is why it is still there. This should be removed. Thanks for pointing out.